test_address_book.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import address_book
+from pathlib import Path
+
+
+class TestAddressBook(unittest.TestCase):
+    def setUp(self):
+        self.john = address_book.Contact("John", "email.address", "123")
+        self.ab = address_book.AddressBook('contact.json')
+        self.ab.add_contact('John', self.john)
+
+    def tearDown(self):
+        Path(self.ab.contactsfile).unlink()
+
+    def test_show(self):
```

Comment:
This test is not for address book it is for contact. You should create another class to test contact